dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -489,7 +490,7 @@ private static void handleBitBucketTokens(DropwizardTestSupport<DockstoreWebserv
                             "update token set content = '" + cachedToken.getContent() + "', dbUpdateDate = '" + cachedToken.getDbUpdateDate().toLocalDateTime().toString() + "' where id = "
                                 + cachedToken.getId());
                     }
-                } catch (IOException | UncheckedIOException e) {
+                } catch (IOException | UncheckedIOException | JsonSyntaxException e) {
```

Comment:
Just curious, why did this change? Seems harmless, especially since it's only in the tests, but still wondering.

Reply:
I cherry-picked this from develop, I think we're using the same bitbucket token cache between hotfix and develop. 
However, the format seems to have changed a little, possibly due to a gson library upgrade, resulting in the exception. 
Since it's an optimization, I didn't look too deeply at it.